resources/js/Pages/Kelas.vue

```diff
@@ -280,30 +280,63 @@
 
       <!-- List All Kelas -->
       <div class="absolute w-full h-full animation-enable"
-          :class="[{ 'left-minFull': !currentPage },
-                  { 'left-0': currentPage }]" v-bar>
+          :class="[{ 'left-minFull': !currentPage},
+                  { 'left-0': currentPage}]" v-bar>
         <div>
           <div class="w-full h-8"/>
           <transition-group name="kelas-list" tag="div">
             <div v-for="kelas in listAllKelas" v-bind:key="kelas.id" 
                 class="animation-enable-short pr-8 hover:pr-4 opacity-75 cursor-pointer z-50 w-full h-16 flex text-center my-4 font-merri-bold text-3xl text-yellow-400"
                 :class="'kelas-'+kelas.id"
                 v-on:click="kelas != chosenKelas ? changeForm(kelas): resetForm(kelas)">
-              <div class="w-3/6 h-full uppercase py-2 bg-green-700">
+              <div class="w-4/12 h-full uppercase py-2 bg-green-700">
                 {{ kelas.kelas }}
               </div>
-              <div class="w-1/6 h-full uppercase py-2 bg-green-600">
+              <div class="w-3/12 h-full uppercase py-2 bg-green-600">
                 {{ kelas.hari }}
               </div>
-              <div class="w-1/6 h-full py-1 bg-green-500 rounded-r-full text-yellow-300">
+              <div class="w-2/12 h-full py-1 bg-green-500 text-yellow-300">
                 {{ kelas.shift }}
               </div>
+              <div class="w-1/12 h-full py-1 bg-white rounded-r-full text-black">
+                {{ kelas.totalGroup }}
+              </div>
             </div>
           </transition-group>
           <div class="w-full h-8"/>
         </div>
       </div>
 
+      <!-- List all Group will continued after this PR
```

Comment:
Ah yeah classy, 

Um I think we can create an entire different branch when working on this feature

Reply:
I think so too, but it was late because I"ll definietly forget what I was doing if deleted :)